src/houston/views/partials/status/review.jade

```diff
@@ -2,6 +2,5 @@ div.fab
   i.fa.fa-user
   div.popover-container
     div.popover-content
-      h2 Reviewing
-      p #{project.name} is being tested by a human for quality assurance.
+      h2 #{project.name} is Being Tested by a Human
       p Sit back and have a drink. We'll let you know if anything is amiss.
```

Comment:
Derp, missed this comment in my review. It's a bit too casual, maybe. Can be addressed in another PR. 😉